robottelo/vm_capsule.py

```diff
@@ -295,16 +291,17 @@ def _setup_capsule(self):
         # manually start pulp_celerybeat service if BZ1446930 is open
         result = self.run('systemctl status pulp_celerybeat.service')
         if 'inactive (dead)' in '\n'.join(result.stdout):
-            if bz_bug_is_open(1446930):
-                result = self.run('systemctl start pulp_celerybeat.service')
-                if result.return_code != 0:
-                    raise CapsuleVirtualMachineError(
-                        'Failed to start pulp_celerybeat service\n{}'.format(
-                            result.stderr)
-                    )
-            else:
+            # BEGIN BZ:1446930
+            result = self.run('systemctl start pulp_celerybeat.service')
+            if result.return_code != 0:
                 raise CapsuleVirtualMachineError(
-                    'pulp_celerybeat service not running')
+                    'Failed to start pulp_celerybeat service\n{}'.format(
+                        result.stderr)
+                )
+            # When BZ is fixed the above can be replaced with the below.
+            # raise CapsuleVirtualMachineError(
```

Comment:
so we already have the code here, but we just going to comment it out so it requires manual intervention once the bug gets closed? like...seriously? :D

Reply:
that is the only I found given the constraint "no more bugzilla calls at test runtime"